lib/eex/lib/eex/engine.ex

```diff
@@ -96,6 +96,9 @@ defmodule EEx.Engine do
 
   All other markers are not implemented by this engine.
   """
+  @spec handle_expr(Macro.t, '' | '=', Macro.t) :: Macro.t
```

Comment:
@josevalim is there a way to defined the second argument as `char_list` and give the option of those two values only?